deploy/base/db/secrets.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql
+type: Opaque
+data:
+  mysql-root-password: "cGFzc3dvcmQ="
```

Comment:
Can we use other more standard garbled ones?

Reply:
I wasn't aware of such other standard. Examples?

Reply:
Nvm, forgot it was 64encoded :facepalm: